themes/src/main/resources-community/theme/base/account/messages/messages_lv.properties

```diff
@@ -0,0 +1,197 @@
+# encoding: utf-8
+doSave=Saglab\u0101t
```

Comment:
Looks like you haven't changed the encoding to UTF-8, and the values are still using ISO-8859-1, there are various tools that can help you with transforming it automatically to UTF-8.